home/.config/home-manager/home-github-actions.nix

```diff
@@ -0,0 +1,6 @@
+{ ... }:
+
+{
+  imports = [ ./common.nix ];
+  home.username = "runner";
```

Comment:
This will be used in https://github.com/kachick/dotfiles/issues/219!

Reply:
Can we use https://github.com/nix-community/home-manager/pull/1889/files#diff-d5f57c2acb984cc8780a622fd667126bef36bcca54bc6b6af286380d9ad628fbR424?